downstream/assemblies/platform/assembly-controller-awx-manage-utility.adoc

```diff
@@ -2,8 +2,8 @@
 
 = The _awx-manage_ Utility
 
-The `awx-manage` utility is used to access detailed internal information of {ControllerName}.
-Commands for `awx-manage` must run as the `awx` or `root` user.
+Use the `awx-manage` utility to access detailed internal information of {ControllerName}.
+Commands for `awx-manage` must only run as the `awx` user.
```

Comment:
Let's move the "only" to the end per [Don](https://github.com/ansible/awx/pull/14980#discussion_r1521140376)

Reply:
Done